docs/technical_specs/model_libraries/power_generation/unit_models/boiler_heat_exchanger_3streams.rst

```diff
@@ -0,0 +1,91 @@
+Heat Exchanger With Three Streams
+=================================
+
+.. index::
+   pair: idaes.power_generation.unit_models.heat_exchanger_3streams; HeatExchangerWith3Streams
+
+.. currentmodule:: idaes.power_generation.unit_models.heat_exchanger_3streams
+
+The HeatExchangerWith3Streams model consists of a heat exchanger with three inlets, `side_1` represents the hot stream, while `side_2` and `side_3` are cold streams.
```

Comment:
Given the fairly specific nature of the model, would it be better to call this an air preheater model? Notably, I am think about the fact that hot and cold streams are defined, so it is not so much a "3 stream HX" as a "1 hot 2 cold stream HX".

Reply:
model is too simple to call it air preheater and it can be used for other unit operations